lexos/processors/prepare/scrubber.py

```diff
@@ -625,59 +526,107 @@ def get_punctuation_string() -> str:
-        punctuation_map,
-        open(
-            punctuation_filename,
-            'wb'))  # Cache
+        all_punctuation_map, open(punctuation_filename, 'wb'))  # Cache
 
-    punctuation = "["
-    for key in punctuation_map:
-        punctuation += chr(key)
-    punctuation += " ]"
-    return punctuation
+    # The resulting string looks like "[,.?!@<more punctuation here>^&* ]"
+    all_punctuation_str = "".join(chr(key) for key in all_punctuation_map)
+    punctuation_regex = "[" + all_punctuation_str + " ]"
 
+    return punctuation_regex
 
-def remove_stopwords(text: str, removal_string: str) -> str:
-    """Removes stopwords from the text.
 
-    :param text: A unicode string representing the whole text that is being
-        manipulated.
-    :param removal_string: A unicode string representing the list of stopwords.
-    :return: A unicode string representing the text that has been stripped of
-        the stopwords chosen by the user.
+def split_input_word_string(input_string: str) -> List[str]:
+    """Breaks word string inputs into lists of words.
+
+    Word strings are a series of words delimited by newlines, commas, and
+        periods. The raw input of the stop words field from the browser is
+        one example.
+    :param input_string: A string of words input by the user.
+    :return: A list of the user's string broken up into words.
     """
 
-    splitlines = removal_string.split("\n")
+    input_lines = input_string.split("\n")
 
-    remove_list = []
-    for line in splitlines:
+    input_pieces = []    # list of words and ""
+    for line in input_lines:
         line = line.strip()
         # Using re for multiple delimiter splitting
-        line = re.split('[,. ]', line)
-        remove_list.extend(line)
+        tokens = re.split('[,. ]', line)  # maybe use all punctuation
+        input_pieces.extend(tokens)
+
+    # get rid of empty strings in input_pieces
+    input_words = [word for word in input_pieces if word != '']
 
-    remove_list = [word for word in remove_list if word != '']
+    return input_words
 
-    # Create pattern
+
+def delete_words(text: str, remove_list: List[str]) -> str:
+    """Deletes the words in remove_list from the text.
+
+    :param text: The original text string.
+    :param remove_list: A list of words to be removed from the text.
+    :return: The updated text, containing only words that were not in
+        remove_list.
+    """
+
+    # Create regex pattern
     remove_string = "|".join(remove_list)
     # Compile pattern with bordering \b markers to demark only full words
     pattern = re.compile(r'\b(' + remove_string + r')\b', re.UNICODE)
-    # Swap stopwords for empty string
-    text = pattern.sub('', text)
+    # Swap target words for empty string
+    scrubbed_text = pattern.sub('', text)
 
     # Replace multiple spaces with a single one
-    text = re.sub(' +', ' ', text)
-    return text
+    scrubbed_text = re.sub(' +', ' ', scrubbed_text)
```

Comment:
Does this line belongs to this function? Since this function is called `delete_words`